servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/util/KarafUtils.java

```diff
@@ -500,13 +322,73 @@ public static String getBundleSymbolicName(IModule module) throws CoreException
 	 * @return
 	 */
 	public static String getBundleVersion(IModule module, File f) throws CoreException {
-		String version = null;
 		
-		if (module == null || module.getProject() == null)
+		if (module == null)
 			return "";
 		
-		// retrieve the bundle version
-		version = getManifestValue(module, "Bundle-Version");
+		String version = null;
+		String packaging = getPackaging(module);
+		IFile manifest = module.getProject().getFile("target/classes/META-INF/MANIFEST.MF");
```

Comment:
going into the target folder seems dangerous for me. I think that PDE is providing some tooling for that